resources/dicts/relationship_events/normal_interactions/dislike/increase.json

```diff
@@ -133,14 +246,70 @@
 			"dislike": "increase"
 		}
 	},
+	{
+		"id": "dislike_kit_inc_med2",
+		"interactions": [
+			"m_c purposefully tripped r_c.",
+            "m_c growls at r_c to get out of {PRONOUN/m_c/poss} way.",
+			"m_c snaps at r_c's annoying antics.",
+            "m_c snarls at r_c when asked to play. {PRONOUN/m_c/subject} will not spend {PRONOUN/m_c/poss} time playing stupid games."
+		],
+		"main_status_constraint": [
+			"kitten",
+            "elder",
+			"apprentice",
+			"warrior",
+			"mediator apprentice",
+			"mediator",
+			"medicine cat apprentice",
+			"medicine cat",
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"kitten"
+		],
+        "main_trait_constraint": [
+			"troublesome",
+            "cold",
+            "vengeful",
+            "strange",
+            "grumpy",
+            "bullying",
+            "bossy"
+		],
+		"reaction_random_cat": {
+			"dislike": "increase"
+		}
+	},
 	{
 		"id": "dislike_dep_leader_inc_med1",
 		"interactions": [
 			"m_c punishes r_c with extra work.",
 			"m_c divides r_c into extra patrols.",
+			"m_c tells other cats that r_c is not doing well in {PRONOUN/r_c/poss} training.",
+			 "m_c gave r_c twice as much work as the other cats {PRONOUN/r_c/poss} age."
+		],
+		"main_status_constraint": [
+			"deputy",
+			"leader"
+		],
+		"random_status_constraint": [
+			"apprentice",
+			"warrior"
+		],
+		"reaction_random_cat": {
+			"dislike": "increase"
+		}
+	},
+	{
+		"id": "dislike_dep_leader_inc_high1",
+        "intensity": "high",
+		"interactions": [
+            "m_c pulls r_c aside and reminds {PRONOUN/r_c/object} {PRONOUN/m_c/subject} could easily exile {PRONOUN/r_c/object}.",
+            "m_c scolded r_c for not doing the hard task {PRONOUN/m_c/subject} assigned {PRONOUN/r_c/object} knowing it was a job for more then one cat.",
 			"m_c gives r_c a particularly hard task, expecting {PRONOUN/r_c/object} to fail.",
-			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!",
-			"m_c tells other cats that r_c is not doing well in {PRONOUN/r_c/poss} training."
+			"m_c threatens r_c with never passing {PRONOUN/r_c/poss} assessment!"
```

Comment:
this one needs to be apprentice only, this rel group currently includes warriors

Reply:
Segment for that one is now added